### DISCUSSION TITLE
Tools to Download

### DISCUSSION THREAD

**Discussion by sanaousuke** [pjy612/SteamManifestCache#770] [Question / Answer]
Hi, there is any Tools to download data games from steam server without steamtools or greenluma? the tools so far i founded is using depotcache manifest, username & password. But can't find tools which is using decrypted keys

**Comment by pjy612** [comment]
https://github.com/wxy1343/DepotDownloader

**Reply by sanaousuke** [reply]
Thanks, but the depot is from author or steam server?

**Reply by pjy612** [reply]
> Thanks, but the depot is from author or steam server?

 steam server
